src/dev/flang/fe/FrontEnd.java

```diff
@@ -265,37 +255,23 @@ private LibraryModule module(String m, Path p)
    * @return the loaded module or null if it was not found or an error occurred.
    */
   LibraryModule loadModule(String m)
-  {
-    return loadModule(m, false);
-  }
-  LibraryModule loadModule(String m,
-                           boolean ignoreNotFound // NYI: remove when module support is stable
-                           )
   {
     var result = _modules.get(m);
```

Comment:
I like the method `computeIfAbsent` from the `Map<K,V>` interface for lazily filling a TreeMap.